backend/core/src/seed.ts

```diff
@@ -20,7 +20,11 @@ const newSeed = (): ListingSeed => {
 
 async function bootstrap() {
   const app = await NestFactory.createApplicationContext(SeederModule.forRoot({ test: argv.test }))
+  const userService = app.get<UserService>(UserService)
   const listing1 = await seedListing(app, newSeed())
+  listing1.leasingAgents.forEach((agent: User) => {
+    void userService.confirm({ token: agent.confirmationToken })
```

Comment:
confirm is an async action and doing it in a foreach loop will start the execution of each confirmation but will not await for it to complete, use `await Promise.all([])` syntax for it.